layouts/brukeropplevelse-og-datadeling/seksjoner/list.html

```diff
@@ -0,0 +1,24 @@
+{{ define "main" }}
+<div>
+  {{ with .Site.GetPage "/brukeropplevelse-og-datadeling/seksjoner" }}
```

Comment:
Hardcoded list. Is this ok?

Reply:
Not sure if I understand. Is there another way to do this?

Reply:
Yeah, you could get all children and use a generic layout that coould be reused. But this is ok for now, we can fix later :)